Library/Formula/libwebm.rb

```diff
@@ -0,0 +1,29 @@
+require 'formula'
```

Comment:
This line isn't needed. Please run `brew audit --strict libwebm` and fix all warnings, thanks!